src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -131,7 +167,7 @@ public override HttpClient GetObserverClient()
         /// <returns></returns>
         private async Task<string> Get(string path)
         {
-            var request = new HttpRequestMessage(HttpMethod.Get, $"https://support-bay-api.azurewebsites.net/observer/{path}?api-version=2.0");
```

Comment:
>}?api-version=2.0" [](start = 123, length = 18)

Hey Cindy, I'm pretty sure you changed this because the GET postbody calls are failing with a 400 when adding the api-version. I can fix this part on my side so that you don't need to do this.

Reply:
Thanks a lot Hawk! That will be great! I will revert this back after your changes:)